doc/rpc/v08/starknet_api_openrpc.json

```diff
@@ -1,7 +1,7 @@
 {
   "openrpc": "1.0.0-rc1",
   "info": {
-    "version": "0.8.0-rc.0",
```

Comment:
Hmm, I don't see an "official" 0.8.0-rc.2 release yet in the repo. Maybe we should update this only once there's an official release?

Or shall this file always contain the actual specs we're implementing (but even then, I'd just keep the version as is on upstream so that we can easily diff).